card-component-app/viewUser.jsx

```diff
@@ -0,0 +1,25 @@
+const ViewUser = () => {
+  const [userId, setUserId] = React.useState(1);
+  return (
+    <div className='search'>
+      <form>
+        <label>Users</label>
+        <select>
```

Comment:
Really cool idea for the search interface. Instead of using an onClick event handler for the option, you want to use an onChange event for the select box itself. https://reactjs.org/docs/events.html#form-events

Reply:
```suggestion
        <select onChange={(event) => setUserId(event.target.value) }>
```

The `event` target in this case will be the option that you just selected, so for the value you can store just the id number instead of `'User ${userId}'` to make things easier.